cisco_aci/datadog_checks/cisco_aci/tags.py

```diff
@@ -9,66 +9,113 @@
 from . import helpers
 from . import exceptions
 
+# TODO create constrant for json attr that are used often
+TN_REGEX = re.compile('/tn-([a-zA-Z-_0-9]+)/')  # TODO '/tn--([^/]+)/'  ?
+APP_REGEX = re.compile('/ap-([a-zA-Z-_0-9]+)/')  # TODO '/ap-([^/]+)/'  ?
```

Comment:
these are more compact ways of doing it, I think it's fine to do that they look pretty nice